### PR TITLE
Couple of simple detections

### PR DESCRIPTION
**Pull request by dicolanl**
Fixes #

## Proposed Changes
  - New TI Detection (IP) against App Services HTTP Logs
  - New App Services AV Detections based on new feature (https://azure.github.io/AppService/2020/12/09/AzMon-AppServiceAntiVirusScanLogs.html)

### CODE REVIEW

File: Detections/ThreatIntelligenceIndicator/IPEntity_AppServiceHTTPLogs.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  | extend TI_ipEntity = iff(isempty(TI_ipEntity) and isnotempty(NetworkSourceIP), NetworkSourceIP, TI_ipEntity)
+  | extend TI_ipEntity = iff(isempty(TI_ipEntity) and isnotempty(EmailSourceIpAddress), EmailSourceIpAddress, TI_ipEntity)
+  | join (
+      AppServiceHTTPLogs | where TimeGenerated >= ago(dt_lookBack)
+    | where isnotempty(CIp)
+    | extend WebApp = split(_ResourceId, '/')[8]
+    // renaming time column so it is clear the log this came from
+    | extend AppService_TimeGenerated = TimeGenerated
+  )
+  on $left.TI_ipEntity == $right.SingleIP
```
Comment (shainw) [inline Detections/ThreatIntelligenceIndicator/IPEntity_AppServiceHTTPLogs.yaml:43]: There is no SingleIP from the right side (AppServiceHTTPLogs) so the join fails, this looks like it should be CIp.

File: Detections/AzureAppServices/AVScan_Infected_Files_Found.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 1
+query: |
+
+  let timeframe = ago(1d);
+  AppServiceAntivirusScanAuditLogs
+  | where NumberOfInfectedFiles > 0
+  | extend HostCustomEntity = _ResourceId
```
Comment (shainw) [inline Detections/AzureAppServices/AVScan_Infected_Files_Found.yaml:19]: Please add - extend timestamp = TimeGenerated

File: Detections/AzureAppServices/AVScan_Failure.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 1
+query: |
+
+  let timeframe = ago(1d);
+  AppServiceAntivirusScanAuditLogs
+  | where ScanStatus == "Failed"
+  | extend HostCustomEntity = _ResourceId
```
Comment (shainw) [inline Detections/AzureAppServices/AVScan_Failure.yaml:19]: Please add - extend timestamp = TimeGenerated

**Review by shainw — CHANGES_REQUESTED** [review]

**Comment by dicolanl** [comment]
@shainw  i updated the 3 files per feedback.  Thanks!

**Review by shainw — APPROVED** [review]
There is no issue with this, the validation needs to be fixed, but I am going to wait to complete it until we have a chance to look at the validation.

**Review by dicolanl — COMMENTED** [review]
fixed

**Review by shainw — APPROVED** [review]

**Pull request merged**